service/ziti-tunnel/service/ipc.go

```diff
@@ -489,15 +489,17 @@ func serveIpc(conn net.Conn) {
 		if cmdErr := dec.Decode(&cmd); cmdErr == io.EOF {
 			break
 		} else if cmdErr != nil {
-			log.Fatal(cmdErr)
+			log.Error(cmdErr)
+			respondWithError(enc, "could not decode command properly", UNKNOWN_ERROR, cmdErr)
+			break
```

Comment:
you don't want to break here right? wouldn't we want to continue instead? breaking will exit the ipc loop, right? we don't want that at this time.

Reply:
probably line 490 needs a continue too instead of break?

Reply:
actually line 490 needs a respondWithError on it too or the IPC reader might be stuck forever waiting for us to reply

Reply:
done. Please review again